packages/jaeger-ui/src/model/ddg/transformTracesToPaths.test.js

```diff
@@ -15,7 +15,14 @@
 import transformTracesToPaths from './transformTracesToPaths';
 
 describe('transform traces to ddg paths', () => {
-  const makeSpan = (spanName, childOf) => ({
+  const makePath = (pathSpans, trace) => ({
+    path: pathSpans.map(({ processID, operationName: operation }) => ({
+      service: trace.data.processes[processID].serviceName,
+      operation,
+    })),
+    attributes: [{ key: 'exemplar_trace_id', value: trace.data.traceID }],
+  });
+  const makeSpan = (spanName, childOf, kind) => ({
```

Comment:
`childOf` is a poor name. What is passed is a `parent`.

Reply:
good point, updated